flake.nix

```diff
@@ -0,0 +1,39 @@
+{
+  inputs = {
+    nixpkgs.follows = "tullia/nixpkgs";
+    std.follows = "tullia/std";
+    tullia.url = github:input-output-hk/tullia;
+  };
+
+  outputs = {
+    self,
+    std,
+    tullia,
+    ...
+  } @ inputs:
+    std.growOn {
+      inherit inputs;
+      cellsFrom = ./nix;
+      cellBlocks = [
+        (std.functions "library")
+        (std.functions "hydraJobs")
+        (tullia.tasks "pipelines")
+        (std.functions "actions")
+      ];
+    }
+    (
+      tullia.fromStd {
+        actions = std.harvest self ["cloud" "actions"];
+        tasks = std.harvest self ["automation" "pipelines"];
+      }
+    )
+    {
+      hydraJobs = std.harvest self ["automation" "hydraJobs"];
```

Comment:
Do we need this? I think this PR should be about _moving_ the Hydra-CI stuff to Cicero. So feel free to remove all the now legacy things (release.nix, this `hydraJobs` expression) and make things less indirected